php.ini-development

```diff
@@ -1348,21 +1339,72 @@ session.cookie_domain =
 
 ; Whether or not to add the httpOnly flag to the cookie, which makes it inaccessible to browser scripting languages such as JavaScript.
 ; http://php.net/session.cookie-httponly
-session.cookie_httponly =
+session.cookie_httponly = 1
 
 ; Handler used to serialize data.  php is the standard serializer of PHP.
 ; http://php.net/session.serialize-handler
 session.serialize_handler = php
 
+; Defines session TTL (Time To Live) value in seconds for session internal data.
```

Comment:
Shouldn't it just be `for session data`? When reading `session internal data` I have the array data of `$_SESSION['__PHP_SESSION__']` in mind.